onnxmltools/convert/keras/operator_converters/Conv.py

```diff
@@ -4,7 +4,14 @@
 # license information.
 # --------------------------------------------------------------------------
 import numpy
-from keras.layers import Conv1D, Conv2D, Conv3D, Conv2DTranspose, Conv3DTranspose, DepthwiseConv2D
+from keras.layers import Conv1D, Conv2D, Conv3D, Conv2DTranspose, Conv3DTranspose
+
+try:
+    from keras.layers import DepthwiseConv2D
+except ImportError:
+    # keras <= 2.1.2
```

Comment:
According to the keras release note, DepthwiseConv2D  was introduced in Keras 2.1.5

Reply:
We should maybe reduce the number of versions we can support. The last one for sure. The number of downloads would help choosing the others I guess.